build.gradle

```diff
@@ -1,12 +1,29 @@
+group 'com.marklogic'
+version '0.0.1'
+
 /*
  * Just run "gradle" from the command line and the script will package of the build into a zip file in the build
  * directory.
  */
 apply plugin: 'java'
+apply plugin: 'application'
+
+repositories {
+	jcenter()
+	maven { url "http://repo.spring.io/milestone" }
+	maven { url "http://developer.marklogic.com/maven2/" }
+	mavenLocal()
+}
+
+dependencies {
+	compile "org.json:json:20160810"
+	compile "com.marklogic:marklogic-xcc:8.0.6"
+	compile "com.marklogic:marklogic-spring-batch-core:0.7.4"
+}
+
 
-defaultTasks 'clean', 'buildRowbot'
 
-def targetDir = "build/rowbot"
+defaultTasks 'clean', 'distZip'
```

Comment:
The distZip gradle task will package your application into the build/ directory.  It's not a fat jar but it will generate a lib folder with your dependencies and some start scripts.  What is missing is the Oracle jar.  That will need to be added later.